pr_testing/test_multiple_prs.sh

```diff
@@ -689,11 +689,25 @@ if [ "X$DO_STATIC_CHECKS" = "Xtrue" -a "X$CMSSW_PR" != X -a "$RUN_TESTS" = "true
   cp -R $WORKSPACE/$CMSSW_IB/llvm-analysis/*/* $WORKSPACE/llvm-analysis || true
   if $IS_DEV_BRANCH && [ $(grep ': error: ' $WORKSPACE/llvm-analysis/runStaticChecks.log | wc -l) -gt 0 ] ; then
     echo "EDM_ML_DEBUG_CHECKS;ERROR,Static Check build log,See Log,llvm-analysis/runStaticChecks.log" >> ${RESULTS_DIR}/static.txt
+  elif $IS_DEV_BRANCH && [ $(grep ': warning: ' $WORKSPACE/llvm-analysis/runStaticChecks.log | grep edm::eventsetup::EventSetupRecord::get | wc -l) -gt 0 ] ; then
+    echo "EDM_ML_DEBUG_CHECKS;WARNING,Static Check build log,See Log,llvm-analysis/runStaticChecks.log" >> ${RESULTS_DIR}/static.txt
   else
     echo "EDM_ML_DEBUG_CHECKS;OK,Static Check build log,See Log,llvm-analysis/runStaticChecks.log" >> ${RESULTS_DIR}/static.txt
   fi
   echo 'END OF STATIC CHECKS'
   echo '--------------------------------------'
+
+  echo 'CMSCODE_CHECKS;OK,CMS Code checks outputs,See Static Checks,llvm-analysis' >> ${RESULTS_DIR}/cmscode.txt
+  echo '--------------------------------------'
+  #USER_LLVM_CHECKERS='-enable-checker threadsafety.ESRecordGetChecker' scram b -k -j8 checker 2>&1 | tee sa-checker.txt
+  USER_CXXFLAGS='-Wno-register -DEDM_ML_DEBUG -w' SCRAM_IGNORE_PACKAGES="Fireworks/% Utilities/StaticAnalyzers" USER_CODE_CHECKS_ARGS='-checks=-*,cms*' scram b -k -j ${NCPU2} code-checks-run SCRAM_IGNORE_SUBDIRS=test 2>&1 | tee -a $WORKSPACE/llvm-analysis/runCMSCodeChecks.log
```

Comment:
@gartung , `USER_CODE_CHECKS_ARGS` is to pass extra args to clang-tidy. This is not going to override the checks 
https://github.com/cms-sw/cmssw-config/blob/scramv3/SCRAM/GMake/Makefile.coderules#L15
https://github.com/cms-sw/cmssw-config/blob/scramv3/SCRAM/GMake/Makefile.coderules#L25

I think you need to set `USER_CODE_CHECKS='cms*'`

Reply:
If you look at the code checks log, USER_CODE_CHECKS_ARGS worked although not as you expected.
 I need to explicitly set checks=-*,cms* to disable the other checks and only get warnings from cms checks.

Reply:
I see how USER_CODE_CHECKS is used now so I made your suggested change.